packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -334,7 +334,11 @@ class FlutterError extends AssertionError {
 ///
 /// The `maxFrames` argument can be given to limit the stack to the given number
 /// of lines. By default, all non-filtered stack lines are shown.
-void debugPrintStack({ int maxFrames }) {
+///
+/// A `label` argument can be given to have the label printed before the stack.
```

Comment:
[This is really nit picky, so sorry].  You could just say
 +/// If the `label` isn't null then it's printed before the stack.